src/Compilers/CSharp/Test/Symbol/DocumentationComments/MethodDocumentationCommentTests.cs

```diff
@@ -125,6 +126,12 @@ public void TestParams()
         }
 
         [Fact]
+        public void TestTuples()
+        {
+            Assert.Equal("M:Acme.Widget.M7((System.Int32,System.Int16))", _widgetClass.GetMembers("M7").Single().GetDocumentationCommentId());
```

Comment:
❗️ The documentation comment ID for `M7` should be identical to the documentation comment for the following method `M8`, except that it will have the word `Value` in it:

```csharp
public void M8(Tuple<int, short> z) { }
```